---
title: C6103 | Microsoft Docs
ms.date: 11/15/2016
ms.prod: visual-studio-dev14
ms.technology: vs-ide-code-analysis
ms.topic: reference
f1_keywords:
- C6103
ms.assetid: 22d1ab35-31a3-4ba9-8ef4-7a64bce66621
caps.latest.revision: 6
author: corob-msft
ms.author: corob
manager: jillfra
ms.openlocfilehash: 5dcc42a19d55351853e7dd53e4184dd0b9b997b9
ms.sourcegitcommit: 68f893f6e472df46f323db34a13a7034dccad25a
ms.translationtype: MT
ms.contentlocale: pt-BR
ms.lasthandoff: 02/15/2020
ms.locfileid: "77277711"
---
# <a name="c6103"></a>C6103
[!INCLUDE[vs2017banner](../includes/vs2017banner.md)]

aviso C6103: retornando \<variável > da chamada de função com falha na linha \<local >.  
  
 Um caminho bem-sucedido por meio da função está retornando uma variável usada como um parâmetro `_Out_` para uma chamada de função interna malsucedida.  
  
 O problema talvez seja a função chamada e a função de chamada não anotadas por completo. A função de chamada pode exigir `_Always_`, `_Outptr_result_nullonfailure_` (`_COM_Outptr_` para código COM), ou uma anotação relacionada, e a função de chamada pode exigir uma anotação `_Success_`. Outra causa possível para esse aviso é que a anotação `_Success_` na função chamada está incorreta.  
  
## <a name="see-also"></a>Consulte Também  
 [Usando anotações de SAL para reduzir defeitos de código do C/C++](../code-quality/using-sal-annotations-to-reduce-c-cpp-code-defects.md)
